service-rpc/src/main/java/org/apache/iotdb/rpc/RpcUtils.java

```diff
@@ -51,6 +51,12 @@
    */
   public static final int FRAME_HARD_MAX_LENGTH = 536870912;
 
+  /**
+   * if resizeIfNecessary is called continuously with a small size for more than
+   */
```

Comment:
The comment is incomplete.

Reply:
It seems to miss something, I'll supply for it